ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -145,7 +145,7 @@ tests:
     test:
     - ref: windows-e2e-operator-test
     workflow: openshift-e2e-vsphere-ccm-ovn-hybrid-install
-- as: azure-e2e-upgrade-from-eus-4.16
+- as: azure-e2e-upgrade-from-previous-eus
```

Comment:
I see, the `.` is the beauty here. 

What about using `azure-e2e-upgrade-from-eus-4-16`



@saifshaikh48   4.16 is not technically the prev, given the job is targeting 4.18.

The path must be is strict 4.16 -> master (4.18), after the 4.19 branching day this job cannot go into master, it must stay in 4.18 branch 


this

Reply:
correct